reference/api/tasks.md

```diff
@@ -6,6 +6,152 @@ The `/tasks` route gives information about the progress of [asynchronous operati
+
+#### `indexUpdate`
+
+| Name         | Description                                                                                                |
+|--------------|------------------------------------------------------------------------------------------------------------|
+| `primaryKey` | Value of the `primaryKey` field . `null` if no `primaryKey` is specified at the time of the index update   |
+
+#### `indexDeletion`
+
+| Name               | Description                                                                           |
+|--------------------|---------------------------------------------------------------------------------------|
+| `deletedDocuments` | Number of deleted documents. This should equal all the documents in the deleted index |
+
+#### `settingsUpdate`
+
+| Name                   | Description                   |
+|------------------------|-------------------------------|
+| `rankingRules`         | List of ranking rules         |
+| `filterableAttributes` | List of filterable attributes |
+| `distinctAttribute`    | The distinct attribute        |
+| `searchableAttributes` | List of searchable attributes |
+| `displayedAttributes`  | List of displayed attributes  |
+| `sortableAttributes`   | List of sortable attributes   |
+| `stopWords`            | List of  stop words           |
+| `synonyms`             | List of synonyms              |
+| `typoTolerance`        | The `typoTolerance` object    |
+| `pagination`           | The `pagination` object       |
+| `faceting`             | The `faceting` object         |
+
+#### `dumpCreation`
+
+| Name      | Description                     |
+| -------   | ------------------------------- |
+| `dumpUid` | The generated `uid` of the dump |
+
+### `error`
+
+**Type**: Object
+
+**Description**: Error details and context. Only present when a task has the `failed` status
+
+| Name    | Description                                                                |
+|---------|----------------------------------------------------------------------------|
+|`message`| A human-readable description of the error                                  |
+|`code`   | The [error code](/reference/api/error_codes.md)                            |
+|`type`   | The [error type](/resources/faq.md#what-do-the-different-error-types-mean) |
+|`link`   | A link to the relevant section of the documentation                        |
+
+### `duration`
+
+**Type**: String
+
+**Description**: The total elapsed time the task spent in the processing state, in ISO 8601 format
+
+### `enqueuedAt`
+
+**Type**: String
+
+**Description**: The date and time when the task was first `enqueued`, in RFC 3339 format
+
+### `startedAt`
+
+**Type**: String
+
+**Description**: The date and time when the task began processing, in RFC 3339 format
+
+### `finishedAt`
+
+**Type**: String
+
+**Description**: The date and time when the task finished processing, whether failed or succeeded, in RFC 3339 format
```

Comment:
```suggestion
**Description**: The date and time when the task finished processing, whether `failed` or `succeeded`, in RFC 3339 format
```

Reply:
I put it in backticks everywhere, it makes sense either way